Velaptor/Reactables/Core/IReactable.cs

```diff
@@ -29,6 +29,14 @@ public interface IReactable<T> : IDisposable
         /// <param name="unsubscribeAfterProcessing">Unsubscribes all of the reactors after the notification has been pushed.</param>
         void PushNotification(T data, bool unsubscribeAfterProcessing = false);
 
+        /// <summary>
+        /// Ends notifications by invoking <see cref="Reactor{T}.OnCompleted"/> to all subscribed <see cref="IReactor{T}"/>s.
+        /// </summary>
+        /// <remarks>
+        ///     Will not invoked the <see cref="IReactor{T}"/>.<see cref="IReactor{T}.OnCompleted"/> more than once.
```

Comment:
Removed "d" from invoke.
```suggestion
        ///     Will not invoke the <see cref="IReactor{T}"/>.<see cref="IReactor{T}.OnCompleted"/> more than once.
```